oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -203,7 +203,6 @@ boolean verifyServerIdentity(byte[] serializedServerIdentity,
   @Override
   public void finalize() {
     requestObserver.onCompleted();
-    channel.shutdownNow();
```

Comment:
Does the error occur because the channel is already closed at this point?

Reply:
Yes